### DISCUSSION TITLE
github.com:sign in with a passkey

### DISCUSSION THREAD

**Discussion by Akabeilaa** [community/community#102197] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

When logging in to github.com, there is an option when logging in with a passkey: Use a mobile phone/tablet to log in. Tip: Scan this QR code through the device where the passkey you want to use for github.com is located. How do I scan this QR code to log in?

**Comment by naveen-98** [comment]


**Reply by Akabeilaa** [reply]
grateful! The QR code generated by passkey when logging in seems to be used for two-factor authentication, that is, after logging in through the username and password, use the authenticator on the mobile phone to scan the code to get a dynamic 6-digit verification code, which serves as an enhanced password verification the process of. But I saw a video where you can directly scan the QR code to log in without entering a username and password.

**Reply by queenofcorgis** [reply]
@naveen-98 
We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Comment by Mahitej28** [comment]
Hello @Akabeilaa for this just install google Authenticator and there you will get an option to scan the QR code also apart from this you can also login with the Github password as that alternative option would be available. But my personal suggestion is to go with google authenticator its the best way to keep your account safe.

> ### Select Topic Area
> Question
> 
> ### Body
> When logging in to github.com, there is an option when logging in with a passkey: Use a mobile phone/tablet to log in. Tip: Scan this QR code through the device where the passkey you want to use for github.com is located. How do I scan this QR code to log in?

**Reply by Akabeilaa** [reply]
grateful! The QR code generated by passkey when logging in seems to be used for two-factor authentication, that is, after logging in through the username and password, use the authenticator on the mobile phone to scan the code to get a dynamic 6-digit verification code, which serves as an enhanced password verification the process of. But I saw a video where you can directly scan the QR code to log in without entering a username and password.

**Reply by Mahitej28** [reply]
Yess we can but authenticator works best for not only github but your discord and other applications too

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Akabeilaa ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!
